### PR TITLE
Tweaks and improvements to ui and load screen

### PR DESCRIPTION
**Pull request by danwalmsley**

### CODE REVIEW

File: src/XamlPlayground.Wasm/AppBundle/index.html [hunk 1]
```diff
@@ -2,27 +2,28 @@
 <html>
 
 <head>
-    <title>XamlPlayground</title>
+    <title>Avalonia UI Playground</title>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
-    <base href="/" />
```
Comment (wieslawsoltes) [inline src/XamlPlayground.Wasm/AppBundle/index.html:8]: We need that

File: src/XamlPlayground.Wasm/AppBundle/index.html [hunk 2]
```diff
@@ -2,27 +2,29 @@
 <html>
 
 <head>
-    <title>XamlPlayground</title>
+    <title>Avalonia UI Playground</title>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no" />
```
Comment (wieslawsoltes) [inline src/XamlPlayground.Wasm/AppBundle/index.html:7]: This was in original web blazor to mobile works

File: src/XamlPlayground/Views/HeaderView.axaml [hunk 3]
```diff
@@ -60,104 +103,10 @@
...
-              <Setter Property="Background" Value="Transparent" />
-            </Style>
-            <Style Selector="CheckBox:checked:pointerover /template/ Border#NormalRectangle">
-              <Setter Property="BorderBrush" Value="#7F8b44ac" />
-              <Setter Property="Background" Value="Transparent" />
-            </Style>
-          </CheckBox.Styles>
-        </CheckBox>
-
-        <NumericUpDown Value="{Binding EditorFontSize, Mode=TwoWay}"
```
Comment (wieslawsoltes) [inline src/XamlPlayground/Views/HeaderView.axaml:100]: This was really useful

File: src/XamlPlayground/ViewModels/MainViewModel.cs [hunk 4]
```diff
@@ -38,7 +38,7 @@ public partial class MainViewModel : ViewModelBase
     public MainViewModel()
     {
         _runSubject = new Subject<(string? xaml, string? code)>();
-        _editorFontSize = 12;
+        _editorFontSize = 14;
```
Comment (wieslawsoltes) [inline src/XamlPlayground/ViewModels/MainViewModel.cs:41]: This is unusable on smaller resolutions and mobile

File: src/XamlPlayground/Views/HeaderView.axaml [hunk 5]
```diff
@@ -5,34 +5,77 @@
...
       <Label FontSize="18"
-             Margin="0,6,0,6" 
+             Foreground="White"
+             FontWeight="SemiBold"
+             Margin="6,6,0,6" 
              VerticalAlignment="Center"
              Classes="text"
              Content="Xaml Playground" />
-    </DockPanel>
-    <ScrollViewer Grid.Row="0" Grid.Column="1"
```
Comment (wieslawsoltes) [inline src/XamlPlayground/Views/HeaderView.axaml:24]: Was added so on mobile you can use it

File: src/XamlPlayground/ViewModels/MainViewModel.cs (hunk 4 above)
Comment (danwalmsley) [reply src/XamlPlayground/ViewModels/MainViewModel.cs]: restored.

**Pull request merged**